llvm/test/ObjectYAML/GOFF/GOFF-Header-End.yaml

```diff
@@ -1,18 +1,19 @@
 # RUN: yaml2obj %s | od -v -An -tx1 | FileCheck --ignore-case %s
 
-# Verify that GOFF Header is correct.
+## Verify that GOFF Header is correct.
 # CHECK: 03 f0 00 00 00 00 00 00 00 00 00 00 00 00 00 00
```

Comment:
Nit: it's common to add some extra whitespace between `CHECK:` and the check pattern, to make the pattern line up with the following lines. Applies to both tests.

Reply:
Fixed